### PR TITLE
Database split

### PR DESCRIPTION
**Pull request by noliveleger**
Needs to be merged with:
- `kpi` https://github.com/kobotoolbox/kpi/pull/2222
- `kc` https://github.com/kobotoolbox/kobocat/pull/531
- `kobo-docker` https://github.com/kobotoolbox/kobo-docker/pull/230

Closes: 
- #49
- #48
- #47 
- #28

### CODE REVIEW

File: helpers/config.py [hunk 1]
```diff
@@ -820,8 +822,10 @@ def __questions_postgres(self):
 
         Settings can be tweaked thanks to pgconfig.org API
         """
-        self.__config["postgres_db"] = CLI.colored_input("Postgres database", CLI.COLOR_SUCCESS,
-                                                         self.__config.get("postgres_db"))
+        self.__config["kc_postgres_db"] = CLI.colored_input("KoBoCat PostgreSQL database", CLI.COLOR_SUCCESS,
+                                                         self.__config.get("kc_postgres_db"))
```
Comment (jnm) [inline helpers/config.py]: @noliveleger notes that existing installs will need this to be set to the value of `postgres_db`, so we could try something like `self.__config.get("postgres_db", self.__config.get("kc_postgres_db"))`

File: helpers/setup.py [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 class Setup:
 
-    KOBO_DOCKER_BRANCH = "master"
+    KOBO_DOCKER_BRANCH = "kobo-install-two-databases"
```
Comment (jnm) [inline helpers/setup.py]: Will use `master` by the time we merge this PR

**Comment by almereyda** [comment]
As this is merged, it appears the issues listed after *Closes:* in the pull request description can be closed, too?

**Comment by noliveleger** [comment]
> As this is merged, it appears the issues listed after _Closes:_ in the pull request description can be closed, too?

Indeed, and done.

**Pull request merged**